File: GTTS/INL.UnitLibraryService.Models/GetUnitLocation_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.UnitLibraryService.Models
{
    public class GetUnitLocation_Item : IGetUnitLocation_Item
    {
        public long LocationID { get; set; }
        public string LocationName { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string AddressLine3 { get; set; }
        public int StateID { get; set; }
        public int CityID { get; set; }
        public bool IsActive { get; set; }
        public int ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UnitLibraryService.Models/get-unit-location_item.ts


import { IGetUnitLocation_Item } from './iget-unit-location_item';

export class GetUnitLocation_Item implements IGetUnitLocation_Item {
  
	public LocationID: number = 0;
	public LocationName: string = "";
	public AddressLine1: string = "";
	public AddressLine2: string = "";
	public AddressLine3: string = "";
	public StateID: number = 0;
	public CityID: number = 0;
	public IsActive: boolean = false;
	public ModifiedByAppUserID: number = 0;
  
}


